docs/reference/manifest/action.md

```diff
@@ -59,6 +73,18 @@ Required element when **xsi:type** is "ShowTaskpane". Specifies the source file
 
```

Comment:
Unable to add suggestions for lines 54 and 64. Consider replacing double quotations with backticks for values, "ExecuteFunction" and "ShowTaskpane".